externals/MemoryPoolBlock.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
`Copyright (c) Microsoft`

corefx doesn't have a single `cs` file with `Copyright (c) Microsoft`, everything is `Copyright (c) .NET Foundation.`.